subsys/bluetooth/CMakeLists.txt

```diff
@@ -5,14 +5,17 @@ add_library(subsys__bluetooth INTERFACE)
 target_include_directories(subsys__bluetooth INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
 
 add_subdirectory(common)
+add_subdirectory_ifdef(CONFIG_BT_CONN services)
```

Comment:
Instead of modifying this patch with a `[noup]` on top of a `[temphack]` commit (https://github.com/nrfconnect/sdk-zephyr/commit/2f39f35b1b7d1fbc1813606a435f410b7f74960b), could you please fix this:
https://projecttools.nordicsemi.no/jira/browse/NCSDK-10028

Reply:
The part of this task is already fixed under this PR https://github.com/nrfconnect/sdk-nrf/pull/4984/files#diff-9bca706979b311a3bd4662437968b9fcfc854228b8d342154900ded7959a82f6R11. Should I do a revert commit and next add a new one @tejlmand ?

Reply:
No, when I say revert, I mean this code should be reverted:
```
if (NOT CONFIG_BT_RPC_CLIENT)
  add_subdirectory_ifdef(CONFIG_BT_HCI host)
  add_subdirectory_ifdef(CONFIG_BT_SHELL shell)
  add_subdirectory_ifdef(CONFIG_BT_CONN services)
  add_subdirectory_ifdef(CONFIG_BT_MESH mesh)
  add_subdirectory_ifdef(CONFIG_BT_AUDIO audio)

  if(CONFIG_BT_CTLR AND CONFIG_BT_LL_SW_SPLIT)
    add_subdirectory(controller)
  endif()
```

This PR brings in this addition:
```
else()
  zephyr_library_sources(host/uuid.c)
```
in addition.
This is what I mean by: `modifying this patch with a [noup] on top of a [temphack] commit`.

The `else()` clause you are introducing in this PR is depending on this `if` clause from a `temphack` commit which should be fixed instead of adding more to the mess.

Reply:
Please look at: https://github.com/nrfconnect/sdk-zephyr/commit/2f39f35b1b7d1fbc1813606a435f410b7f74960b